gradle-plugin/build.gradle

```diff
@@ -10,7 +10,7 @@ group = "com.lightbend.akka.grpc"
 // https://github.com/palantir/gradle-git-version/issues/97
 
 def tag = "git describe --tags".execute().text.substring(1).split("-g")[0].replace("\n", "")
-def finalVersion = (tag == versionDetails().lastTag.substring(1)) ? tag : tag.reverse().reverse() + "-" + versionDetails().gitHash.substring(0, 8)
+def finalVersion = (tag == versionDetails().lastTag.substring(1)) ? tag : tag.reverse().reverse() + "-" + versionDetails().gitHash.substring(0, 8) + "-SNAPSHOT"
```

Comment:
Hmmm, always uses a "-SNAPSHOT" suffix in `finalVersion`?

Reply:
If `tag == versionDetails().lastTag.substring(1)` it uses the tag as-is, otherwise it adds the git hash and `-SNAPSHOT` suffix. (not sure what the `.reverse().reverse()` is about btw...)

Reply:
Ah, didn't match up the parenthesises correctly 👍